frontend_tests/node_tests/stream_data.js

```diff
@@ -290,13 +290,11 @@ run_test('is_active', () => {
 
     var sub;
 
+    page_params.demote_inactive_streams = 2;
```

Comment:
Put a comma defining these values in the tests.

Reply:
Actually, never mind, use e.g. `display_settings.demote_inactive_streams_values.automatic.code` here and below.